cmake-docker-environment/labs/consume-shared-library/readme.md

```diff
@@ -0,0 +1,51 @@
+# consume-shared-library
+Commands for generating a SHARED library and consume it from an executable. Before you start, change to the proper directory and make the build directory.
+   ```
+   $ cd consume-shared-library
+   $ mkdir build && cd build
+   ```
+   
+## Exercise 1: Generate and consume a STATIC libary, check its size
+   ```
+   $ cmake -D BUILD_SHARED_LIBS=true ..
```

Comment:
Better use ``BUILD_SHARED_LIBS=ON``, it is more common

Reply:
Updated in new commit.